config/main.py

```diff
@@ -3534,6 +3534,34 @@ def delete(ctx):
 
     sflow_tbl['global'].pop('agent_id')
     config_db.set_entry('SFLOW', 'global', sflow_tbl['global'])
+@config.group(cls=AbbreviationGroup)
+@click.pass_context
+def switch(ctx):
+    """switch related config"""
+    pass
+
+@switch.command('shell')
+@click.pass_context
+@click.argument('en', metavar='<en>', required=True, type=int)
+def set_sdk_diag_shell(ctx, en):
```

Comment:
This approach may not work when SWSS docker is restarting and this command is issued. Is there a way to sync post swss is up ?

Reply:
When SWSS is down syncd also will be down, and setting shell property is not applicable.

It would also result more un wanted complexity